src/PSRule.Rules.Azure/rules/Azure.ContainerApp.Rule.yaml

```diff
@@ -98,6 +98,30 @@ spec:
       - field: properties.vnetConfiguration.infrastructureSubnetId
         hasValue: true
 
+---
+# Synopsis: Use of Azure Files volume mounts to persistent storage container data.
+apiVersion: github.com/microsoft/PSRule/v1
+kind: Rule
+metadata:
+  name: Azure.ContainerApp.Storage
+  ref: AZR-000364
+  tags:
+    release: GA
+    ruleSet: 2023_03
+    Azure.WAF/pillar: Reliability
+spec:
+  level: Warning
```

Comment:
I think we can allow this to default to `Error` now that we are only targeting confirmed storage requirements.

What do you think?

Reply:
Agreed, specially given how the rule works now.